addon/index.js

```diff
@@ -136,7 +147,46 @@ function setupApp() {
       });
     });
 
-    if (self.loadReason === 'install') {
+    if (showInstalledDialog && self.loadReason === 'install') {
+      const installMsgPanel = Panel({ // eslint-disable-line new-cap
+        contentURL: './base.html',
+        contentScriptFile: './link-catch.js',
+        onShow: () => {
+          installMsgPanel.port.emit('show', Mustache.render(templates.installed, {
+            base_url: settings.BASE_URL
+          }));
+        }
+      });
+
+      installMsgPanel.port.on('link', (url) => {
+        const tabExists = false;
+
+        // reload the testpilot tab now that we are logged in with the
+        // addon installed
+        for (let tab of tabs) {
+          if (tab.url.indexOf(`${settings.BASE_URL}`) > -1) {
```

Comment:
Maybe I'm missing something here, but do we need the variable interpolation here, or can we just do something like:
```js
if (tab.url.indexOf(settings.BASE_URL) > -1) {
```

Or if we know we're in a recent version of Firefox, we could just use the new `.includes()` method:
```js
if (tab.url.includes(settings.BASE_URL)) {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes